catalyst/loggers/comet.py

```diff
@@ -0,0 +1,254 @@
+from typing import Dict, List
+import pickle
+
+import numpy as np
+
+from catalyst.core.logger import ILogger
+from catalyst.settings import SETTINGS
+
+if SETTINGS.comet_required:
+    import comet_ml
+
+
+class CometLogger(ILogger):
+    """Comet logger for parameters, metrics, images and other artifacts (videos, audio,
+    model checkpoints, etc.).
+
+    Comet documentation: https://www.comet.ml/docs/
+
+    To start with Comet please check out: https://www.comet.ml/docs/quick-start/.
+    You will need an ``api_key`` to log your Catalyst runs to Comet.
+
+    Args:
+        workspace : Optional, ``str``. Attach an experiment to a project the belongs to this workspace.
+            Read more about workspaces in the `Comet User Interface docs <https://www.comet.ml/docs/user-interface/>`
+        project_name: Optional, ``str``, the name of the project within Comets's run.
+        experiment_id: Optional, pass an Experiment Key string if you want to continue logging to an existing experiment (resume experiment).
+            Read more about Existing Experiment `here <https://www.comet.ml/docs/python-sdk/ExistingExperiment/>`_.
+        tags: Optional, pass a list of tags to add to the Experiment. Tags will be shows in the dashboard.
+        experiment_kwargs: Optional, additional keyword arguments to be passed directly to the
+          `Experiment.__init__() <https://www.comet.ml/docs/python-sdk/Experiment/#experiment__init__>`_ function.
+
+    Python API examples:
+
+    .. code-block:: python
+
+        from catalyst import dl
+
+        runner = dl.SupervisedRunner()
+        runner.train(
+            ...
+            loggers={
+                "comet": dl.CometLogger(
+                    project_name="my-comet-project
+                )
+            }
+        )
+
+    .. code-block:: python
+
+        from catalyst import dl
+
+        class CustomRunner(dl.IRunner):
+            # ...
+
+            def get_loggers(self):
+                return {
+                    "console": dl.ConsoleLogger(),
+                    "comet": dl.CommetLogger(
+                        project_name="my-comet-project"
+                    )
+                }
+            # ...
+
+        runner = CustomRunner().run()
+    """
+
+    def __init__(
+        self,
+        workspace: str = None,
+        project_name: str = None,
+        experiment_id: str = None,
+        comet_mode: str = "online",
+        tags: List = None,
+        logging_frequency: int = 10,
```

Comment:
I would change the default here to 1.

Reply:
Me and Doug thought it would be better if the default value would be a value that doesn't cause the throttle.